openfl/utilities/utils.py

```diff
@@ -26,6 +28,36 @@ def getfqdn_env(name: str = '') -> str:
     return getfqdn(name)
 
 
+def is_fqdn(hostname: str) -> bool:
+    """https://en.m.wikipedia.org/wiki/Fully_qualified_domain_name."""
+    if not 1 < len(hostname) < 253:
+        return False
+
+    # Remove trailing dot
+    hostname.rstrip('.')
+
+    #  Split hostname into list of DNS labels
+    labels = hostname.split('.')
+
+    #  Define pattern of DNS label
+    #  Can begin and end with a number or letter only
+    #  Can contain hyphens, a-z, A-Z, 0-9
+    #  1 - 63 chars allowed
+    fqdn = re.compile(r'^[a-z0-9]([a-z-0-9-]{0,61}[a-z0-9])?$', re.IGNORECASE) # noqa 
```

Comment:
Could you please specify error code in NOQA?

Reply:
@dmitryagapov https://github.com/MichaelKim0407/flake8-use-fstring/issues/7